testing/factory/cluster_test.go

```diff
@@ -53,7 +53,7 @@ var _ = Describe("Cluster", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(cluster).NotTo(BeNil())
 			cluster.WithDefaults()
-			Expect(cluster.Spec.Replicas).NotTo(Equal(uint32(2)))
+			Expect(cluster.Spec.Replicas).NotTo(Equal(int32(2)))
 			Expect(cluster.Spec.Port).NotTo(Equal(uint16(3306)))
```

Comment:
I think this changes should be reverted but if not this test should be changed as well.